addons/website_event_meet/static/src/js/snippets/options.js

```diff
@@ -4,6 +4,28 @@ import options from 'web_editor.snippets.options';
 
 options.registry.WebsiteEvent.include({
 
+    /**
+     * @override
+     */
+    async start() {
+        const res = await this._super(...arguments);
+        this.currentWebsiteUrl = this.ownerDocument.location.pathname;
+        this.eventId = this._getEventObjectId();
+        // Only need for one RPC request as the option will be destroyed if a
+        // change is made.
+        const rpcData = await this._rpc({
+            model: 'event.event',
+            method: 'read',
+            args: [
+                [this.eventId],
+                ['meeting_room_allow_creation'],
+            ],
+        });
+        this.data.reload = this.currentWebsiteUrl;
```

Comment:
I don't believe this is needed anyway. Seems like we used to set this after the RPC because we would change the reload url to something we got from the RPC.

Since it's already set in the `super`, no need to set it again after this RPC